dace/sdfg/validation.py

```diff
@@ -55,6 +58,46 @@ def validate_sdfg(sdfg: 'dace.sdfg.SDFG'):
                     "storage type. Please use a different storage location." %
                     name, sdfg, None)
 
+            #Check for valid bank assignments
+            try:
+                bankassignment = sdutil.parse_location_bank(desc)
+            except ValueError as e:
+                raise InvalidSDFGError(
+                    f"Failed to parse location['bank'] for "
+                    f" array {name}: {str(e)}", sdfg, None)
```

Comment:
Don't catch and rethrow, just let the exception propagate (or if you insist, at least copy the error message directly from `e`).